transit_odp/timetables/extract.py

```diff
@@ -164,6 +179,17 @@ def extract(self) -> ExtractedData:
             operating_profiles=operating_profiles,
         )
 
+    def extract_flexible_journey_details(self):
+        services = self.doc.get_services()
+        flexible_journey_patterns = flexible_journey_patterns_to_dataframe(services)
+        if not flexible_journey_patterns.empty:
+            flexible_journey_patterns["file_id"] = self.file_id
+            flexible_journey_patterns.set_index(
+                ["file_id", "journey_pattern_id"], inplace=True
+            )
+            return flexible_journey_patterns
```

Comment:
We can move this outside the if block and remove the other return of empty dataframe

Reply:
done